.gitattributes

```diff
@@ -0,0 +1,15 @@
+* text=auto
+
+*.conf text eol=lf
+*.sh text eol=lf
+*.cfg text eol=lf
+*.properties text eol=lf
+*.xml text eol=lf
+*.html text eol=lf
+*.jsx text eol=lf
+*.js text eol=lf
+*.tsx text eol=lf
+*.ts text eol=lf
+*.css text eol=lf
+*.sass text eol=lf
+*.less text eol=lf
```

Comment:
har pensjon bruk for crlf noen steder?
Burde *.yml, *.yaml, *.md, *.adoc også tas med?

Reply:
Det var jeg som fikk masse CRLF warns fra obsolete frontend ...  Slette katalogene så gikk det bra